fdbclient/include/fdbclient/MetaclusterManagement.actor.h

```diff
@@ -1759,6 +1759,15 @@ struct ConfigureTenantImpl {
 		     ++configItr) {
 			if (configItr->first == "tenant_group"_sr) {
 				wait(updateTenantGroup(self, tr, self->updatedEntry, configItr->second));
+			} else if (configItr->first == "assigned_cluster"_sr &&
+			           configItr->second != tenantEntry.get().assignedCluster) {
+				auto& newClusterName = configItr->second;
+				TraceEvent("UpdateManagementCluster")
+				    .detail("OriginalAssignedCluster",
+				            tenantEntry.get().assignedCluster.present() ? tenantEntry.get().assignedCluster.get()
+				                                                        : "null"_sr)
```

Comment:
You can log an optional directly and it will print an appropriate string if the optional is not set:

```suggestion
				            tenantEntry.get().assignedCluster)
```